src/api/app/controllers/webui/request_controller.rb

```diff
@@ -515,6 +502,16 @@ def staging_status(request, target_project)
     }
   end
 
+  def cache_diff_data
+    return @diff_not_cached = true unless defined?(@action)
+
+    return unless (@diff_not_cached = @action[:diff_not_cached])
```

Comment:
I'm beginning to wonder why we need this extra variable if we have `@action[:diff_not_cached]`...

Reply:
You triggered one of my _hot buttons_ so ----> https://github.com/openSUSE/open-build-service/pull/14312/commits/4d7b597629c5fbdad0708dcd5b1c557be012ae23